src/WebUI/Services/ServicesTable.tsx

```diff
@@ -63,11 +67,18 @@ export class ServicesTable extends BaseComponent<IServicesComponentProperties> {
 
     private _openServiceItem = (event: React.SyntheticEvent<HTMLElement>, tableRow: ITableRow<any>, selectedItem: IServiceItem) => {
         if (selectedItem) {
-            ActionsHubManager.GetActionsHub<SelectionActions>(SelectionActions).selectItem.invoke({ item: selectedItem, showSelectedItem: true, selectedItemType: SelectedItemKeys.ServiceItemKey });
+            const payload: ISelectionPayload = { 
+                item: selectedItem, 
+                itemName: (selectedItem.service as V1Service).metadata.name,
+                showSelectedItem: true, 
+                selectedItemType: SelectedItemKeys.ReplicaSetKey 
+            };
+
+            this._selectionActionCreator.selectItem(payload);
         }
     }
 
-    private static _getServiceItems(serviceList: V1Service[]): IServiceItem[] {
+    public static _getServiceItems(serviceList: V1Service[]): IServiceItem[] {
```

Comment:
public functions without _, and if needed we can move this to util class.
if possible keep this on top of all private functions, easy readability

Reply:
Done. This is a hack right now. Rajendra is working to make this class work with V1Service object just like the other component which also work off of kube objects directly and not custom objects